### DISCUSSION TITLE
Proxy connection headers not checked for as standard

### DISCUSSION THREAD

**Discussion by pettzilla1** [encode/httpcore#925] [General]
https://github.com/encode/httpcore/blob/c96fdf7d31c3ab983f4eec4ef7a25cd81edc7ac9/httpcore/_async/http_proxy.py#L280
Hi I've noticed that headers which can change during proxy auth aren't checked for to see if they are in the request, I think it may be worth adding a check to see if ('proxy-authorization','proxy-connection','connection') are in the request and need to be added to the proxy headers,
If this can already be done happy to be pointed in the right direction

**Comment by lovelydinosaur** [comment]
I'll bump this into "discussion" as a starting point, as I don't think it's yet clearly framed enough to warrant an issue.
